fem/nonlinearform_ext.hpp

```diff
@@ -25,7 +25,7 @@ class NonlinearFormExtension : public Operator
    NonlinearForm *n; ///< Not owned
 public:
    NonlinearFormExtension(NonlinearForm *form);
-   virtual void AssemblePA() = 0;
+   virtual void Setup() = 0;
```

Comment:
In #1731, this method is renamed to `Assemble`. We need to unify these.

Reply:
I will rename this method to `Assemble` too then.